04week/loop.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+// Create a new file called loops.js in the /04week folder of your workbook.
+// Complete each of the following exercises.
+// for loop
+
+// Use a for loop to console.log each item in the array carsInReverse.
+const carsInReverse = ['Ford', 'Chevy', 'Tesla', 'Nissan', 'Toyota'];
+for (i = 0; i < carsInReverse.length; i++) {
+  console.log(carsInReverse[i]);
+}
+
+// Create an object (an array with keys and values) called persons with the following data:
+// firstName: "Jane"
+// lastName: "Doe"
+// birthDate: "Jan 5, 1925"
+// gender: "female"
+const persons = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female"
+};
+
+// Use a for...in loop to console.log each key.
+for (const keys in persons) {
+  console.log(Object.keys(persons));
+}
+
+// Then use a for...in loop and if state to console.log the value associated with the key birthDate.
+for (const keys in persons) {
+  if (keys === 'birthDate') {
+    console.log(persons[keys]);
+  }
+}
+
+// Use a while loop to console log numbers 1 to 1000
+let num = 1;
```

Comment:
You define num as 1, which is fine, and you iterate through it in your wile loop, but then in your do loop, num is already 1000, so the do loop only runs once. Just reset num back to 1 after your while loop!